shell/pages/c/_cluster/uiplugins/index.vue

```diff
@@ -0,0 +1,634 @@
+<script>
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+
+import { DEV } from '@shell/store/prefs';
+import { sortBy } from '@shell/utils/sort';
+import { allHash } from '@shell/utils/promise';
+import { CATALOG, UI_PLUGIN, SCHEMA } from '@shell/config/types';
+import { CATALOG as CATALOG_ANNOTATIONS } from '@shell/config/labels-annotations';
+
+import ActionMenu from '@shell/components/ActionMenu';
+import Tabbed from '@shell/components/Tabbed/index.vue';
+import Tab from '@shell/components/Tabbed/Tab.vue';
+import IconMessage from '@shell/components/IconMessage.vue';
+import UninstallDialog from './UninstallDialog.vue';
+import InstallDialog from './InstallDialog.vue';
+import DeveloperInstallDialog from './DeveloperInstallDialog.vue';
+import PluginInfoPanel from './PluginInfoPanel.vue';
+import SetupUIPlugins from './SetupUIPlugins';
+import RemoveUIPlugins from './RemoveUIPlugins';
+import { isUIPlugin, uiPluginHasAnnotation, UI_PLUGIN_NAMESPACE } from '@shell/config/uiplugins';
+
+export default {
+  components: {
+    ActionMenu,
+    DeveloperInstallDialog,
+    IconMessage,
+    InstallDialog,
+    PluginInfoPanel,
+    Tab,
+    Tabbed,
+    UninstallDialog,
+    SetupUIPlugins,
+    RemoveUIPlugins,
+  },
+
+  data() {
+    return {
+      view:              '',
+      charts:            [],
+      installing:        {},
+      errors:            {},
+      plugins:           [], // The installed plugins
+      helmOps:           [], // Helm operations
+      loading:           true,
+      menuTargetElement: null,
+      menuTargetEvent:   null,
+      menuOpen:          false,
+    };
+  },
+
+  layout: 'plain',
+
+  async fetch() {
+    const hash = {};
+
+    if (this.hasPluginCRD) {
+      hash.plugins = this.$store.dispatch('management/findAll', { type: UI_PLUGIN });
+    }
+
+    hash.load = await this.$store.dispatch('catalog/load');
+
+    if (this.$store.getters['management/schemaFor'](CATALOG.OPERATION)) {
+      hash.helmOps = await this.$store.dispatch('management/findAll', { type: CATALOG.OPERATION });
+    }
+
+    const res = await allHash(hash);
+
+    this.plugins = res.plugins || [];
+    this.helmOps = res.helmOps || [];
+
+    const c = this.$store.getters['catalog/rawCharts'];
+
+    this.charts = Object.values(c);
+
+    this.loading = false;
+  },
+  computed: {
+    ...mapGetters({ uiplugins: 'uiplugins/plugins' }),
+    ...mapGetters({ uiErrors: 'uiplugins/errors' }),
+
+    menuActions() {
+      const menuActions = [];
+      const isDeveloper = this.$store.getters['prefs/get'](DEV);
+
+      if (isDeveloper) {
+        menuActions.push({
+          action:  'devLoad',
+          label:   this.t('plugins.devloper.label'),
```

Comment:
typo on `devloper`. Should be `developer` as for `devloper` isn't even present on the translation file